ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -516,18 +560,19 @@ open class NavigationBar: UINavigationBar, TokenizedControlInternal {
 
     func updateColors(for navigationItem: UINavigationItem?) {
         let color = navigationItem?.navigationBarColor(fluentTheme: tokenSet.fluentTheme)
+        let shouldHideRegularTitle: Bool = (style == .gradient) && showsLargeTitle
```

Comment:
Nit: do we need ()?

Reply:
Nope just added them to make it slightly easier to read the conditions around the = operators.